### DISCUSSION TITLE
Include "latest" attribute in release object

### DISCUSSION THREAD

**Discussion by prryplatypus** [community/community#64736] [Actions]
### Select Topic Area

Product Feedback

### Body

Hello,

I'm opening this feature request to request that a `latest` attribute is added to the release object in GitHub Action events, so that it's possible to know whether the following checkbox was checked or not when a "release created" workflow is triggered.

![imagen](https://github.com/community/community/assets/25409753/cf346a6b-89ab-4d27-857a-9d46cb51629d)

This would allow us to know whether the release is a release for the latest version of our project (and therefore if we should tag the Docker image with `latest`), or whether it's a patch for a previous LTS version.

Many thanks!

**Comment by prryplatypus** [comment]
Related discussion: https://github.com/orgs/community/discussions/64734

**Comment by mwaddell** [comment]
This would be very helpful for our CI/CD pipeline.  We want to keep one of our deployment slots always matching the "latest" release (for our QA team to use as a baseline for testing).  We have a github action that looks for the release created/updated event that is published/non-draft.  However, this doesn't work if we deploy a new release without that is not "latest" or if we want to switch "latest" to the previous release because we need to roll back our environment.

If this cannot be added to the event payload, would it be possible to have github automatically create/recreate a "latest" tag so that we can monitor for changes to that instead?

**Comment by heruan** [comment]
This would simplify our workflows _a lot_ especially when publishing Docker images where we need to know if the release is marked as `latest` to tag the image as such.

**Comment by paololazzari** [comment]
Shocking that `reactions` is included and this isn't
